<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>设置男孩年龄</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="BoyspinBox"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="BoyIncBtn">
         <property name="text">
          <string>Boy长大一岁</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="MetaInfoBtn">
         <property name="text">
          <string>类的元对象信息</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>设置女孩年龄</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="GirlspinBox"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="GirlIncBtn">
         <property name="text">
          <string>Girl长大一岁</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="ClearBtn">
         <property name="text">
          <string>清空对话框</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ClearBtn</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
